supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesRuntimeAttributesSpec.scala

```diff
@@ -204,7 +253,8 @@ class JesRuntimeAttributesSpec extends WordSpecLike with Matchers with Mockito {
     ()
   }
 
-  private def assertJesRuntimeAttributesFailedCreation(runtimeAttributes: Map[String, WdlValue], exMsg: String, workflowOptions: WorkflowOptions = emptyWorkflowOptions): Unit = {
+  private def assertJesRuntimeAttributesFailedCreation(runtimeAttributes: Map[String, WdlValue], exMsg: String,
+                                                       workflowOptions: WorkflowOptions = emptyWorkflowOptions): Unit = {
```

Comment:
ToL If you do it for one, best to make everything newlined (i.e. move down`exMsg`)